source/command-line-options.txt

```diff
@@ -33,18 +35,77 @@ Use the following syntax to start |compass| and set a configuration option:
 Options
 -------
 
-You can set the following configuration options using the command line:
+You can set these options when using the command line:
+
+.. option:: --autoUpdates
+  
+   Enable automatic updates.
+  
+.. option:: --enableFeedbackPanel
+  
+   Enable the feedback panel.
+   
+.. option:: --enableMaps
+ 
+   Enable geographic visualizations.
+
+.. option:: --exportConnections
+    
+   Export :ref:`favorite connections 
+   <compass-export-import-cli-landing>`.
+
+.. option:: --file <file>
+  
+   Specify a file containing a list of connections. These connections
+   are automatically available once |compass| starts.
+
+.. option:: --help
+   
+   Show |compass| options.
+
+.. option:: --importConnections
+
+   Import :ref:`favorite connections 
+   <compass-export-import-cli-landing>`.
+
+.. option:: --lgDarkmode
+
+   Use leafygreen dark mode instead of darkreader.
+
+.. option:: --networkTraffic
+
+   Configure |compass| to not perform outgoing 
+   :ref:`network operations <compass-configure-network-traffic>`
+   other than those to the database.
+
+.. option:: --passphrase
+ 
+   Connection export/import passphrase.
+
+.. option:: --positionalArguments
+
```

Comment:
This one seems strange. Judging by the description, 'postionalArguments' probably isn't an option flag. These look like positional parameters.

Reply:
It strikes me as odd as well. It looks like these two might actually be for Chromium/Electron. I've removed these.

```
 ./MongoDB\ Compass\ Beta --positionalArguments 
Type for option "positionalArguments" mismatches: expected array, received boolean (while validating preferences from: Command line)
```

```
--ignoreAdditionalCommandLineFlags (*)     Allow Additional CLI Flags
--positionalArguments                      Specify a Connection String or Connection ID to Automatically Connect
```